apps/box/src/screens/Users/Users.screen.tsx

```diff
@@ -0,0 +1,48 @@
+import {
+  FxBox,
+  FxButton,
+  FxSafeAreaBox,
+  FxSpacer,
+  FxText,
+} from '@functionland/component-library';
+import React from 'react';
+import { Image, ScrollView } from 'react-native';
+import { CopyIcon } from '../../components';
+import Clipboard from '@react-native-clipboard/clipboard';
+
+export const UsersScreen = () => {
+  return (
+    <FxSafeAreaBox marginHorizontal="20" flex={1}>
+      <ScrollView>
+        <FxSpacer height={56} />
+        <UserHeader />
+      </ScrollView>
+    </FxSafeAreaBox>
+  );
+};
+
+const UserHeader = () => {
+  const did = 'key:z6Mkhsmw8...';
+
+  const copyHandler = () => {
+    Clipboard.setString(did);
+  };
+
+  return (
+    <FxBox alignItems="center">
+      <Image source={require('./img/sample.png')} />
```

Comment:
Should image not have styling with border radius? I know the sample picture is already round but we don't cater for square pictures

Reply:
I'll update this in subsequent PR when I add support for selecting images.